Win10_VirtualDesktop_Optimize.ps1

```diff
@@ -26,16 +26,37 @@
 ######################################################################################################################################>
 
 [Cmdletbinding()]
-Param
-(
-    [Parameter()]
-    [ValidateSet('1909','2004')]
-    $WindowsVersion = 2004,
+Param (
+    # Parameter help description
```

Comment:
Love that all of the tasks are broken out, but we need a way with one switch that will run them all, possibly in a second parameter set

Reply:
Yeah, I agree.  There should be a default param set for all the tasks vs picking and choosing each task.